Assets/Scripts/ArenaManagerScript.cs

```diff
@@ -41,7 +42,7 @@ public class ArenaManagerScript : MonoBehaviour {
     public AudioClip tickUp;
     public AudioClip tickDown;
 
-    private AudioSource audio;
+    private new AudioSource audio;

```

Comment:
What's the 'new' about?

Reply:
For all these AudioSource properties I was getting a message about the class overwriting an `audio` property from their parent class. I guess that `new` indicates that you're meaning to override an existing property.